packages/namegenerator-sdk/src/index.ts

```diff
@@ -0,0 +1,144 @@
+import fetch from "cross-fetch";
+
+const DEFAULT_ENDPOINT = "http://100.24.45.225/";
+
+export const NameGeneratorGroupingCategory = {
```

Comment:
@lightwalker-eth 

This definition is not required: in the docs, this is defined as "string". I wanted to list all possible grouping categories by using this strategy, which can be removed if you think it will be messy to update this in the future based on NameGenerator API updates.

Reply:
<img width="429" alt="Captura de Tela 2024-12-03 às 14 06 46" src="https://github.com/user-attachments/assets/96a81b83-db40-4e13-a8a2-f1e9ab0b1743">

For reference